tests/includes/juju.sh

```diff
@@ -276,7 +276,9 @@ juju_bootstrap() {
 
 	command="juju bootstrap ${base} ${cloud_region} ${name} --model-default mode= ${BOOTSTRAP_ADDITIONAL_ARGS}"
 	# keep $@ here, otherwise hit SC2124
-	${command} "$@" 2>&1 | OUTPUT "${output}"
+	# We re-run the command through the bash interprutter here to avoid some
+	# weird shell expansion behaviour when using --config.
```

Comment:
```suggestion
	# We run the command through the bash interpreter here to avoid some
	# weird shell expansion behaviour when using --config and other arguments
	# with white space.
```

Reply:
Done & Done